clients/cloud/scala/src/main/scala/io/confluent/examples/clients/scala/Producer.scala

```diff
@@ -0,0 +1,26 @@
+package io.confluent.examples.clients.scala
+
+import java.util.Properties
+import com.typesafe.config.{Config, ConfigFactory}
+import org.apache.kafka.clients.producer.{KafkaProducer, ProducerRecord}
+
+object Producer extends App {
+  var conf = ConfigFactory.load("ccloud.properties")
```

Comment:
file and topic from command line arguments.

Reply:
done